app/models/admin_create_auction.rb

```diff
@@ -0,0 +1,51 @@
+class AdminCreateAuction < Struct.new(:params)
+  attr_reader :auction
+
+  def perform
+    @auction = Auction.create(attributes)
+  end
+
+  private
+
+  def attributes
+    {
```

Comment:
Same question applies from my other comment on this PR. `issue_url` is not listed in this hash, but perhaps it should?